sct-commons/src/test/java/org/lfenergy/compas/sct/commons/scl/ied/InputsAdapterTest.java

```diff
@@ -305,15 +326,4 @@ void filterDuplicatedExtRefs_should_not_remove_not_duplicated_extrefs() {
                 .hasSize(6);
     }
 
-    @Test
-    void updateAllExtRefIedNames_when_DOI_Mod_and_DAI_stVal_notExists_should_not_produce_error() {
-        // Given
-        SCL scl = SclTestMarshaller.getSCLFromFile("/scd-refresh-lnode/Test_Missing_ModstVal_In_LN0_when_binding.scd");
```

Comment:
Test file `/scd-refresh-lnode/Test_Missing_ModstVal_In_LN0_when_binding.scd` can be deleted

Reply:
done